azure-pipelines.vrt-pr.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
```

Comment:
## [🕵 FluentUIV7 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29147&commitId=906801b0d4bc1b66e75ec02f00bedae40cb0d607&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Compound</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound.Root.default.chromium.png|0| Added|
|Button Compound.Root.hover.chromium.png|0| Added|
|Button Compound.Root.pressed.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v7"/>